src/status_im2/contexts/chat/home/chat_list_item/view.cljs

```diff
@@ -208,16 +208,40 @@
      {:color color
       :size  :medium}]))
 
+(defn notification
+  [{:keys [muted group-chat unviewed-messages-count unviewed-mentions-count]}]
+  (let [customization-color (rf/sub [:profile/customization-color])
+        unread-messages?    (> unviewed-messages-count 0)
+        unread-mentions?    (> unviewed-mentions-count 0)]
+    [rn/view {:style style/notification-container}
+     (cond
+       muted
+       [icons/icon :i/muted {:color colors/neutral-40}]
+
+       (and group-chat unread-mentions?)
+       [quo/info-count
+        {:style               {:position :relative :right 0}
+         :customization-color customization-color}
+        unviewed-mentions-count]
```

Comment:
I see duplocation between this case and the one at line 232, Can we add their checks in an `(or ` function?
Or make the default of the cond?

Reply:
Oh no, they are not duplicated:

this one is:
```clojure
(and group-chat unread-mentions?)
       [quo/info-count
        {:style               {:position :relative :right 0}
         :customization-color customization-color}
        unviewed-mentions-count]
```

the other one:
```cojure
unread-messages?
       [quo/info-count
        {:style               {:position :relative :right 0}
         :customization-color customization-color}
        unviewed-messages-count]
```

The differences are the conditions: 
- `unread-messages?` vs `unread-mentions?`
-  `unviewed-mentions-count` vs `unviewed-messages-count`

the thing they share is the `info-count` component.

Also, the default case is `nil`, that happens when we have no notifications (so we don't draw a muted icon, a gray dot or a counter).

WDYT?

Reply:
Can we extract this into a var?
```clojure
[quo/info-count
 {:style               {:position :relative :right 0}
  :customization-color customization-color}
```

Reply:
I also thought about creating this as a separate component, but I think it's not worth it.

- That `info-count` is overriding its styles, but nothing important, just technical debt.
- If I create a separate component, it will just look as a wrapper, it will not add meaningful view code.
- This will be fixed when https://github.com/status-im/status-mobile/issues/16611 is solved.

So right now, I think is better to keep it as is.